DuckDuckGo/MainViewController.swift

```diff
@@ -83,14 +83,14 @@ class MainViewController: UIViewController {
     }()
     
     var homeViewController: HomeViewController?
-    var homeTabViewController: HomeTabViewController?
-    var homeController: HomeViewControllerProtocol? {
-        homeViewController ?? homeTabViewController
+    var newTabPageViewController: NewTabPageViewController?
+    var homeController: NewTabPage? {
```

Comment:
I left `homeController` here deliberately to avoid extensive renaming. It'll be done after the old code gets cleaned up.